File: src/main/java/com/enphoria/geojsoningestor/GeojsonIngestorApplication.java
package com.enphoria.geojsoningestor;

import com.enphoria.geojsoningestor.utils.Init;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class GeojsonIngestorApplication extends Init {

	public static void main(String[] args) {
		SpringApplication.run(GeojsonIngestorApplication.class, args);
	}

}

File: src/main/kotlin/com/enphoria/geojsoningestor/GeojsoningestorApplication.kt
package com.enphoria.geojsoningestor


import com.enphoria.geojsoningestor.transform.Transform2
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
class GeojsoningestorApplication

fun main(args: Array<String>) {
    runApplication<GeojsoningestorApplication>(*args)
    val transform: Transform2 = Transform2()
    transform.readGeojson()

}
